### PR TITLE
Fix networkd config causing connectivity issues

### PR DESCRIPTION
**Pull request by nevalla**

### CODE REVIEW

File: lib/kontena/machine/aws/cloudinit.yml [hunk 1]
```diff
@@ -33,6 +33,8 @@ coreos:
         DOMAINS=kontena.local
```
Comment (SpComb) [inline lib/kontena/machine/aws/cloudinit.yml]: This should be titlecase `Domains=...`

> Nov 09 11:58:37 ... systemd-networkd[1184]: [/run/systemd/network/00-eth.network:8] Unknown lvalue 'DOMAINS' in section 'Network'

From a different host

**Comment by SpComb** [comment]
LGTM but unable to test

Worth fixing the `[Network] Domains=` bit at the same time, to test and make sure `getent hosts foo-1` resolves `foo-1.kontena.local` as expected?

**Comment by nevalla** [comment]
Rebooted a node dozen of times and always got a healthy result. Ship it?
ping @jakolehm

**Comment by jakolehm** [comment]
@nevalla dns works after that change?

**Comment by nevalla** [comment]
```
$ getent hosts loadbalancer-1.kontena.local
10.81.1.159     loadbalancer-1.kontena.local
```

**Comment by jakolehm** [comment]
LGTM

**Pull request merged**